widget/popup_menu.go

```diff
@@ -95,6 +105,15 @@ func (p *PopUpMenu) ShowAtPosition(pos fyne.Position) {
 	p.Show()
 }
 
+// ShowAtRelativePosition shows the pop-up menu at the position relative to given object.
+//
+// Since 2.4
+func (p *PopUpMenu) ShowAtRelativePosition(rel fyne.Position, to fyne.CanvasObject) {
+	withRelativePosition(rel, to, func(pos fyne.Position) {
+		p.ShowAtPosition(pos)
+	})
```

Comment:
The same simplification can be applied here.
```suggestion
	withRelativePosition(rel, to, p.ShowAtPosition)
```